lynxkite-app/web/src/Directory.tsx

```diff
@@ -29,23 +29,53 @@ function EntryCreator(props: {
   onCreate: (name: string) => void;
 }) {
   const [isCreating, setIsCreating] = useState(false);
+  const [error, setError] = useState("");
+
+  function validateName(name: string): boolean {
+    if (name.includes("/")) {
+      setError("Name cannot contain '/' characters");
+      return false;
+    }
+    if (name.trim() === "") {
+      setError("Name cannot be empty");
+      return false;
+    }
+    setError("");
+    return true;
+  }
```

Comment:
Sounds good to me! I think this is in line with what Linux accepts as filename. (Linux also disallows the `null` character (`chr(0)`), but I don't think there's a risk of anyone typing that.)